src/dfx/src/lib/models/canister_id_store.rs

```diff
@@ -1,11 +1,11 @@
 use crate::lib::environment::Environment;
-use crate::lib::error::canister_id_store::StructuredFileOrFilesystemError;
 use crate::lib::error::CanisterIdStoreError;
 use crate::lib::network::directory::ensure_cohesive_network_directory;
 use dfx_core::config::model::dfinity::Config;
 use dfx_core::config::model::network_descriptor::{NetworkDescriptor, NetworkTypeDescriptor};
 
 use candid::Principal as CanisterId;
+use dfx_core::error::unified_io::UnifiedIoError;
```

Comment:
Please move this import to the section above

Reply:
nice catch, thanks